### PR TITLE
Use GitHub actions for automated testing

### PR DESCRIPTION
**Pull request by dlenski**
Removes `.travis.yml` and replaces it with `.github/workflows/test.yml`.

Also adds a Build Status badge, [the image of which](https://github.com/dtcooper/python-fitparse/workflows/test/badge.svg) won't be available _until actually merged_, since running that workflow in this repo will create it.

### CODE REVIEW

File: requirements-test.txt [hunk 1]
```diff
@@ -1,12 +1,2 @@
-#
-# This file is autogenerated by pip-compile
-# To update, run:
-#
-#    pip-compile --output-file requirements-test.txt etc/requirements-test.in
-#
-
-click==7.0                # via pip-tools
```
Comment (dlenski) [inline requirements-test.txt:8]: `click`, `pip-tools,` and `six` were not actually being used in any of the tests, so I removed them here.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+on: [ push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['2.7', '3.3', '3.4', '3.5', '3.6', '3.x', 'pypy2', 'pypy3']
```
Comment (dlenski) [inline .github/workflows/test.yml:14]: Intended to match the versions supported in the old Travis config.

**Comment by dlenski** [comment]
Results of the workflow actions included here (passing :smile:) can be seen at https://github.com/dlenski/python-fitparse/actions?query=workflow%3Atest

**Comment by pR0Ps** [comment]
Amazing, thank you!

**Pull request merged**